build_tools/cmake/build_android.sh

```diff
@@ -50,7 +50,6 @@ cd build
   -DANDROID_ABI="$ANDROID_ABI" \
   -DANDROID_PLATFORM=android-29 \
   -DIREE_BUILD_COMPILER=OFF \
-  -DIREE_BUILD_TESTS=OFF \
```

Comment:
We've been keeping all the options in the build scripts explicit, so maybe make this an explicit ON rather than deleting it?

Reply:
Ah, right. Made it explicit.